libs/external-db-dynamodb/src/sql_exception_translator.ts

```diff
@@ -1,14 +1,20 @@
 import { errors } from '@wix-velo/velo-external-db-commons'
+import { Item } from '@wix-velo/velo-external-db-types'
 const { CollectionDoesNotExists, DbConnectionError } = errors
 
-export const notThrowingTranslateErrorCodes = (err: any) => {
+export const notThrowingTranslateErrorCodes = (err: any, collectionName?: string, metaData?: { items?: Item[] }) => {
     switch (err.name) {
         case 'ResourceNotFoundException':
-            return new CollectionDoesNotExists('Collection does not exists')
+            return new CollectionDoesNotExists('Collection does not exists', collectionName)
         case 'CredentialsProviderError':
             return new DbConnectionError('AWS_SECRET_ACCESS_KEY or AWS_ACCESS_KEY_ID are missing')
         case 'InvalidSignatureException':
             return new DbConnectionError('AWS_SECRET_ACCESS_KEY or AWS_ACCESS_KEY_ID are invalid')
+        case 'TransactionCanceledException':
+            if (err.message.includes('ConditionalCheckFailed')) {
+                const itemId = metaData?.items?.[err.CancellationReasons.findIndex((reason: any) => reason.Code === 'ConditionalCheckFailed')]._id
+                return new errors.ItemAlreadyExists('Item already exists', collectionName, itemId)
```

Comment:
you already destructuring error classes from `errors` so just add `ItemAlreadyExists` to line 3 and then use it